dev/tasks-dev.ts

```diff
@@ -63,7 +63,7 @@ const tasks: Record<string, DenoTaskDefArgs> = {
             "-f",
             file,
           ])
-        } down --remove-orphans`;
+        } down --remove-orphans --volumes`;
```

Comment:
fyi: https://github.com/zifeo/substantial/pull/2/files#r1685540051

Reply:
Nice, thanks. I actually added that in a futile attempt to solve this issue with the bitnami containers and coder. Whenever I restart the coder workspace, the old containers seem to leave "something" around that prevents new ones from working. Specifically, they leave files that are under the wrong owner that prevents the new containers from working.

Why is "something" in scare quotes? Well, these containers don't have any volumes attached. What's more, the conflict happens even after I `compose down` and `compose up` again. Even with the `--volumes` flag. It's as if the images got corrupted but I'm not sure. 

The only solution I've found is to `docker system prune -a` and clean out everything.